frontend/src/components/elements/DataFrame/DataFrame.test.tsx

```diff
@@ -119,14 +133,36 @@ describe("DataFrame Element", () => {
   it("adds extra width for vertical scrollbar when tall but not wide", () => {
     // Be careful to ensure that the number of digits needed to display the
     // largest row number is the same for the two DataFrames.
-    let props = getProps({ ...fakeData(11, 1) })
+    let props = getProps({ ...fakeInt64Data(11, 1) })
     let wrapper = shallow(<DataFrame {...props} />)
     const normalWidth = wrapper.find("MultiGrid").props().width
 
-    props = getProps({ ...fakeData(99, 1) })
+    props = getProps({ ...fakeInt64Data(99, 1) })
     wrapper = shallow(<DataFrame {...props} />)
     const widthWithScrollbar = wrapper.find("MultiGrid").props().width
 
     expect(widthWithScrollbar).toBe(normalWidth + SCROLLBAR_SIZE)
   })
+
+  it("should render string column with text-align set to left", () => {
```

Comment:
one small thing: would recommend also adding a test validating that `node.props().style.textAlign` is not left when `fakeInt64Data` is used for the dataframe data

Reply:
Hi @vdonato, the default value for `node.props().style?.textAlign` is undefined when `fakeInt64Data` is used. Do you think we should check if `.toBeUndefined()` or `.not.toBe("left")`?

Reply:
Either one seems equally okay to me, so I think keeping the tests as-is and checking `.toBeUndefined()` is fine.